integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -311,6 +312,7 @@ func BuildOCR3ConfigForCCIPHome(
 				MaxReportTransmissionCheckAttempts: 5,
 				MaxMerkleTreeSize:                  merklemulti.MaxNumberTreeLeaves,
 				SignObservationPrefix:              "chainlink ccip 1.6 rmn observation",
+				RMNEnabled:                         os.Getenv("ENABLE_RMN") == "true", // only enabled in manual test
```

Comment:
I don't know if there is precedent inside of this integration-tests module, but in core all of our env vars are prefixed with `CL_`